extractor/BUILD.bazel

```diff
@@ -20,21 +20,44 @@ load(
     ":extractor.bzl",
     "extractor_test",
 )
+load("@build_environment//:configuration.bzl", "sdk_version")
 
-TEST_DARS = [
-    "RecordsAndVariants",
-    "PrimitiveTypes",
-    "TransactionExample",
-]
-
-[
-    daml_compile(
-        name = darmod,
-        srcs = ["src/test/resources/damls/%s.daml" % darmod],
-        target = lf_version_configuration.get("default"),
-    )
-    for darmod in TEST_DARS
-]
+genrule(
+    name = "test-dar",
+    srcs = [
+        "//daml-script/daml:daml-script.dar",
+        ":src/test/resources/damls/EnumMod.daml",
+        ":src/test/resources/damls/GenMapMod.daml",
+        ":src/test/resources/damls/PrimitiveTypes.daml",
+        ":src/test/resources/damls/RecordsAndVariants.daml",
+        ":src/test/resources/damls/TransactionExample.daml",
+    ],
+    outs = ["test.dar"],
+    cmd = """
+      set -eou pipefail
+      TMP_DIR=$$(mktemp -d)
+      mkdir -p $$TMP_DIR/daml
+      cp -L $(location :src/test/resources/damls/EnumMod.daml) $$TMP_DIR/daml
+      cp -L $(location :src/test/resources/damls/GenMapMod.daml) $$TMP_DIR/daml
+      cp -L $(location :src/test/resources/damls/RecordsAndVariants.daml) $$TMP_DIR/daml
+      cp -L $(location :src/test/resources/damls/PrimitiveTypes.daml) $$TMP_DIR/daml
+      cp -L $(location :src/test/resources/damls/TransactionExample.daml) $$TMP_DIR/daml
+      cp -L $(location //daml-script/daml:daml-script.dar) $$TMP_DIR/
+      cat << EOF > $$TMP_DIR/daml.yaml
+sdk-version: {sdk}
+name: test
+source: daml
+version: 0.0.1
+dependencies:
+  - daml-stdlib
+  - daml-prim
+  - daml-script.dar
+EOF
+      $(location //compiler/damlc) build --project-root=$$TMP_DIR --ghc-option=-Werror -o $$PWD/$(location test.dar)
+      rm -rf $$TMP_DIR
+    """.format(sdk = sdk_version),
+    tools = ["//compiler/damlc"],
+)
```

Comment:
Looks like there is no easy way of building a DAR with script as a dependency, I copied this and adapted it from the Daml Script tests.

Reply:
totally fine for now, I’ll look into cleaning this up later